metadata.json

```diff
@@ -104,22 +96,42 @@
       "name": "simp/pupmod",
       "version_requirement": ">= 7.1.0 < 8.0.0"
     },
+    {
+      "name": "simp/resolv",
+      "version_requirement": ">= 0.1.1 < 1.0.0"
+    },
     {
       "name": "simp/rsync",
       "version_requirement": ">= 6.0.0 < 7.0.0"
     },
     {
-      "name": "simp/rsyslog",
-      "version_requirement": ">= 7.0.0 < 8.0.0"
+      "name": "simp/selinux",
+      "version_requirement": ">= 2.3.0 < 3.0.0"
     },
     {
-      "name": "simp/simpcat",
+      "name": "simp/simp_openldap",
       "version_requirement": ">= 6.0.0 < 7.0.0"
     },
+    {
+      "name": "simp/simp_rsyslog",
+      "version_requirement": ">= 0.3.0 < 1.0.0"
+    },
     {
       "name": "simp/simplib",
       "version_requirement": ">= 3.12.0 < 4.0.0"
     },
+    {
+      "name": "simp/ssh",
+      "version_requirement": ">= 6.5.1 < 7.0.0"
+    },
+    {
+      "name": "simp/svckill",
+      "version_requirement": ">= 3.3.1 < 4.0.0"
+    },
+    {
+      "name": "simp/swap",
+      "version_requirement": ">= 0.1.2 < 1.0.0"
+    },
     {
       "name": "simp/tftpboot",
```

Comment:
I see that simp::tftpboot is included by simp::server::kickstart, by default.  However, simp::server::kickstart, itself, is not included by default.  If we are excluding simp/sudosh because it is not used by default, seems like we should *not* be including simp::tftpboot as well.  Our conventions are not clear to me.